LICENSE.md

```diff
@@ -0,0 +1,21 @@
+# MIT License
+
+Copyright (c) 2019 phstemplates authors
```

Comment:
Might want to update this to PHS, also related would be to add PHS as a copyright holder in the description. e.g. https://github.com/Public-Health-Scotland/phsmethods/pull/104/commits/986886491b1f1cd2e4c449f10ab832fca9c219f3

Reply:
Changes are slightly random as I was separating out changes not affected by `lib2git` or `git2r`. But you're right that it will be cleaner if we separate out further so will do that soon thanks

Reply:
`run_on()` is used in other functions that were part of the larger pull request - again, I'll separate out into different PRs